com.stansassets.scene-management/BuildConfigurator/Editor/EditorBuildSettingsValidator.cs

```diff
@@ -0,0 +1,40 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace StansAssets.SceneManagement.Build
+{
+    [InitializeOnLoad]
+    public class EditorBuildSettingsValidator
+    {
+        public const string ScenesSyncDescription = "Scenes in the Scene Management do not match the scenes " +
+                                                    " in the Build Settings" +
+                                                    ", they may not work correctly in the editor!";
```

Comment:
Current Editor Build Settings are our of sync with the Scene Management build configuration.